guides/common/modules/ref_storage-guidelines.adoc

```diff
@@ -17,8 +17,8 @@ As {ProjectName} has many operations that are I/O intensive, using high latency,
 Ensure your installation has a speed in the range 60 - 80 Megabytes per second.
 
 ifdef::satellite[]
-You can use the `fio` tool to get this data.
-See the Red{nbsp}Hat Knowledgebase solution https://access.redhat.com/solutions/3397771[Impact of Disk Speed on Satellite Operations] for more information on using the `fio` tool.
+You can use the `storage-benchmark` script to get this data.
+See the Red{nbsp}Hat Knowledgebase solution https://access.redhat.com/solutions/3397771[Impact of Disk Speed on Satellite Operations] for more information on using the `storage-benchmark` script.
```

Comment:
```suggestion
For more information on using the `storage-benchmark` script, see https://access.redhat.com/solutions/3397771[Impact of Disk Speed on {Project} Operations].
```

Reply:
Wording looks better, but "Impact of Disk Speed on Satellite Operations" is a name of the article. Should I add `if satellite`?